api/sbom.proto

```diff
@@ -317,6 +317,15 @@ message ExternalReference {
   }
 }
 
+// Serializer message encapsulates additional metadata related to the original SBOM document.
+message Serializer {
+  string format = 1; // The original format of the SBOM document (e.g., CycloneDX, SPDX).
+  string hash = 2; // The original hash of the SBOM document to ensure integrity.
+  int64 size = 3; // The original size of the SBOM document in bytes.
+  string path = 4; // The original file path of the SBOM document.
+  string uri = 5; // The original URI of the SBOM document.
+}
```

Comment:
The linter should require comments above the field names. Also alphabetize the order of messages, so `Serializer` should go before `Tools`.
 
```suggestion
// Serializer message encapsulates additional metadata related to the original SBOM document.
message Serializer {
  // The original format of the SBOM document (e.g., CycloneDX, SPDX).
  string format = 1;

  // The original hash of the SBOM document to ensure integrity.
  string hash = 2;

  // The original size of the SBOM document in bytes.
  int64 size = 3;

  // The original file path of the SBOM document.
  string path = 4;

  // The original URI of the SBOM document.
  string uri = 5;
}
```